### ISSUE TITLE
Solar Neutron Activator production rate error?

### ISSUE THREAD

**Issue by Fatmice** [mekanism/Mekanism#5092]
#### Issue description:
Adding speed upgrade 3x -> 8x speed -> produces 8mb/t
Adding the 4th upgrade -> 16x speed -> produces 32mb/t
...Adding to the 8th upgrade -> 256x speed -> produces 512 mb/t

Not only that, given input then production rate is double input starting with 4th speed upgrade
Ex: input 60 mb/t -> at 8th speed upgrade -> produces 120 mb/t

#### Steps to reproduce:
1. Add speed upgrade 3x
2. Add another speed upgrade, check rate
3. Add till max speed upgrade, check rate

#### Version (make sure you are on the latest version before reporting):
Forge:14.23.4.2705
Mekanism: 1.12.2-9.4.13.349 
Generators and Tools on same version as Mekanism

Here Lithium input is 60 mb/t.  Expected output should be no more than 60 mb/t of tritium.  Data queried per second then divided by 20 to get per tick
![2018-06-18_10 46 04](https://user-images.githubusercontent.com/8956281/41547121-f73875be-72e4-11e8-989f-fa6cc05d1c62.png)
![2018-06-18_10 46 16](https://user-images.githubusercontent.com/8956281/41547122-f750a7a6-72e4-11e8-9673-5e23e1bfd3a4.png)
![2018-06-18_10 46 20](https://user-images.githubusercontent.com/8956281/41547123-f764786c-72e4-11e8-8916-338368fc3452.png)
![2018-06-18_10 46 32](https://user-images.githubusercontent.com/8956281/41547124-f775fc72-72e4-11e8-8785-9223211ce589.png)
![2018-06-18_10 46 38](https://user-images.githubusercontent.com/8956281/41547125-f7969978-72e4-11e8-9848-4f67cd89756b.png)
![2018-06-18_10 46 51](https://user-images.githubusercontent.com/8956281/41547126-f7c4aa2a-72e4-11e8-88e6-01a44b4d152e.png)
![2018-06-18_10 47 01](https://user-images.githubusercontent.com/8956281/41547128-f7e9a6cc-72e4-11e8-83da-8737e49cd7cb.png)
![2018-06-18_10 47 09](https://user-images.githubusercontent.com/8956281/41547129-f8130300-72e4-11e8-8a34-63daf7e80551.png)

**Comment by Fatmice** [comment]
Input was 60 mB/t so output should be 60 mB/t.  The measured output was double that of input.  Script just measure the difference in output tank level after each second, which was 120 mB/t

**Comment by Fatmice** [comment]
The screenshots showed the measurements as I added more speed upgrades to the machine.

At 3x speed upgrade, the output was the same as in put, 8 mB/t.  This made sense since actual input, 60mB/t, was greater than could be processed by the speed of the machine.  It also made sense as the speed of the machine, 8x, should be equal to the output rate of 8mB/t.

At 4x speed upgrade, the output was was 32 mB/t.  This was not equal to the speed of the machine which was 16x, which should have given 16 mB/t.  At this point it is not obvious that the machine is taking input and doubling it as output.

As I added more speed upgrade, to the max of 8, you saw the ramping up of output to 512 mB/t.  Since actual input was still capped at 60 mB/t, the excess buffers eventually depleted so that the machine could only process at the rate of actual input.  At this point, output should be 60 mB/t but instead you saw on the screen output was 120 mB/t.  This implies that the machine was taking in 60 mB/t but putting out 120 mB/t.  This is clearly doubling and also implies that it started at the 4th speed upgrade.

Honestly, I'm fine with such a bug, but thought I should mention it for the sake of "balance" =P
